packages/marko-web-identity-x/browser/profile.vue

```diff
@@ -322,6 +336,10 @@ export default {
       type: String,
       default: null,
     },
+    returnToDelay: {
```

Comment:
This must be defined and passed through in both the `marko` components, and any other components that include the profile (such as Authenticate, Login, and Comment)

Reply:
@solocommand  I did add this to marko.json and added passing it through the marko component if it is sent along with the input.   

As for adding it to the other things mentioned. The commenting & login component do not call or reference the profile component and the auth component is overriding the submit(profile-update) action & forcing a redirect already [line 27](https://github.com/parameter1/base-cms/blob/master/packages/marko-web-identity-x/browser/authenticate.vue)

I don't think this is currently being called in any other spots.

Reply:
@solocommand  I also checked the gated content part which should be using the reload-page-on-submit=true prop which would force the content page to reload on profile submit.  But in theory the user should be intercepted at the authentication step to finish filling out the required fields and then get redirected to the content landing page.  If they did some how get to the content page without a fully filled out profile page this would still reload the page on submission of a full profile.